automation/SetUp.ps1

```diff
@@ -14,4 +14,5 @@ New-VM @VM
 Set-VM -Name $VMName -ProcessorCount 2 -AutomaticCheckpointsEnabled $false
 Set-VMProcessor -VMName $VMName -ExposeVirtualizationExtensions $true
 Set-VMDvdDrive -VMName $VMName -Path $ISOFile
+Add-VMDvdDrive -VMName $VMName -Path "$PSScriptRoot\auto-install.iso" -ControllerNumber 1 -ControllerLocation 1
```

Comment:
Do we require to Set-VMFirmware to configure the first boot device for VM to boot from the DVD drive during the initial setup or installation?

Reply:
Makes sense to explicitly set a boot order and not rely entirely on Hyper-V defaults. Will explore this suggestion